Update multilanguage.md

josepcrespo opened the pull request:
In order to avoid headaches for others trying to implement Voyager's multilingual feature. I thought it would be positive to improve this part of the documentation. Some people had this problem in 2018 and today it is still happening in the current version (1.4).

I found this thread that helped me solve the problem in my own project:

https://github.com/the-control-group/voyager/issues/3172

I hope that you can accept the changes so that you can help others in the future. Awesome project, thanks!

josepcrespo committed:
Update multilanguage.md

In order to avoid headaches for others trying to implement Voyager's multilingual feature. I thought it would be positive to improve this part of the documentation. Some people had this problem in 2018 and today it is still happening in the current version (1.4).

I found this thread that helped me solve the problem in my own project:

https://github.com/the-control-group/voyager/issues/3172

I hope that you can accept the changes so that you can help others in the future. Awesome project, thanks!

fletch3555 commented:
These instructions are actually incorrect.  `artisan config:cache` will actually set the cache, not clear it

MrCrayon commented:
To what @fletch3555 said I'd add that config is not cached by default and should be cached only during deployment process.
If you cached your config you should know that you need to clear cache when you apply a change to config just like you need to clear route cache if you add a route.
If we don't consider this basic knowledge we should add that cache should be cleared if cached everywhere else in documentation that referers to config changes.

josepcrespo commented:
> These instructions are actually incorrect. `artisan config:cache` will actually set the cache, not clear it

Sorry, my bad. I copy/pasted the Artisan description for "config:clear" but wrote the incorrect Artisan command. The correct one should be "config:clear" instead of "config:cache".

josepcrespo commented:
> To what @fletch3555 said I'd add that config is not cached by default and should be cached only during deployment process.
> If you cached your config you should know that you need to clear cache when you apply a change to config just like you need to clear route cache if you add a route.
> If we don't consider this basic knowledge we should add that cache should be cleared if cached everywhere else in documentation that referers to config changes.

Ok, maybe is a basic knowledge for you, but if you are reading the documentation:

1. It's your first time dealing with that problem or,
2. It's been a long time since you did it previously and, probably you don't remember that "basic knowledge".

Documentation is for… well documenting precesses… it seems clear that I'm not the only one confused about this and, instead of reading a simple Artisan command on the official documentation I had to waste time searching on Google and digging on the GitHub repository issues from two years ago…

I've reformulated the sentence, maybe you can reconsider the pull request now.

josepcrespo committed:
Update multilanguage.md

The paragraph has been reformulated in order to be more accurate (it was somehow incorrect as @fletch3555 said in my first pull request).

josepcrespo commented:
Instead of accepting a simple pull request you prefer to just throw a personal opinion and waste your precious time writing three long sentences arguing it's unnecessary for you. Very productive 👍
